src/exports.js

```diff
@@ -21,9 +21,8 @@ const StyleSheet = {
     create(sheetDefinition /* : SheetDefinition */) {
         return mapObj(sheetDefinition, ([key, val]) => {
             return [key, {
-                // TODO(emily): Make a 'production' mode which doesn't prepend
-                // the class name here, to make the generated CSS smaller.
-                _name: `${key}_${hashObject(val)}`,
+                _name: process.env.APHRODITE_KEYS !== 'MINIFIED' ?
```

Comment:
Another thing that we were considering doing is making something like `css(styles.a, styles.b)` generate only a single hash class name, like `_abcdef` instead of combining two hashes like `_abcdef-O_o-_012345`. Not sure if you'd like to try tackling that here as well, but leaving a TODO about it would be nice if not. :)

If we do want to tackle it, we should figure out whether we need to make our hash longer. Currently, our hash is 6 base-36 characters, which means we'd expect collisions once there's ~47k styles on the page. Is that enough? On the largest KA pages, we end up seeing ~500 independent styles, but our website isn't very SPA-y, so the stylesheet gets cleared out pretty often. If we add uppercase letters, that bumps us up to ~240k styles on a page before seeing a collision.